models/fates/R/write.configs.FATES.R

```diff
@@ -68,7 +68,8 @@ write.config.FATES <- function(defaults, trait.values, settings, run.id){
    ncdf4::nc_close(domain.nc)
    
    ## SURF
-   surf.default <- system.file("surfdata_ref.nc",package = "PEcAn.FATES")
+   #surf.default <- system.file("surfdata_ref.nc",package = "PEcAn.FATES")
+   surf.default <- system.file("surfdata_1x1_brazil_16pfts_simyr2000_c160127.nc",package = "PEcAn.FATES")
```

Comment:
not required for this PR, but this is where we need to be able have a user-specified surf.default provided by the settings